ddtrace/contrib/celery/util.py

```diff
@@ -0,0 +1,31 @@
+# Project
+from ddtrace import Pin
+
+# Service info
+APP = 'celery'
+SERVICE = 'celery'
+
+
+def meta_from_context(context):
+    """ helper to extract meta values from a celery context """
+    meta_keys = (
+        'called_directly', 'correlation_id', 'delivery_info', 'eta', 'expires', 'hostname',
```

Comment:
I'd be interested in most of the identifiers (`id/reply_to/correlation_id/task/hostname`), and everything inside of `delivery_info`.

I would say all of the booleans don't really lead to any debugging processes, as they would be either application semantics of widely duplicated results (`is_eager/utc` are configured globally).

Not sure how @clutchski would like to proceed with the nullable values, but I think it would be a good idea to only display the time related metadata if their fields are populated.